UI/EmuScreen.cpp

```diff
@@ -1393,6 +1391,30 @@ void EmuScreen::render() {
 	if (!thin3d)
 		return;  // shouldn't really happen but I've seen a suspicious stack trace..
 
+	// We can still render behind the pause screen.
+	bool paused = screenManager()->topScreen() != this;
+
+	if (paused) {
+		// If we're paused and PauseScreen is transparent (will only be in buffered rendering mode), we just copy display to output.
+		thin3d->BindFramebufferAsRenderTarget(nullptr, { RPAction::CLEAR, RPAction::DONT_CARE, RPAction::DONT_CARE }, "EmuScreen_Paused");
```

Comment:
Hm, should we check here `screenManager()->topScreen()->isTransparent()`?

-[Unknown]

Reply:
yes, oops.